### ISSUE TITLE
Images inside of a list view appear black

### ISSUE THREAD

**Issue by mikebranstein** [NativeScript/theme#92]
When I apply the list-group  and list-group-item classes to a list view and list view item template, images within the item template appear black. Note that I have added the currently undocumented "thumb" class to the image, but the problem persists with or without this class.

Here's the code:

```
<ListView class="list-group" items="{{ items }}" itemTap="onItemTap">
  <ListView.itemTemplate>
    <StackLayout orientation="horizontal" class="list-group-item">
      <Image class="thumb" src="~/images/test.png" />
      <Label text="Some text..." />
    </StackLayout>
  </ListView.itemTemplate>
</ListView>
```

Here's an Android screenshot:
<img width="407" alt="screen shot 2016-10-23 at 9 01 17 pm" src="https://cloud.githubusercontent.com/assets/2053909/19631016/e1dfd600-9963-11e6-96f7-bb56787f4a0e.png">

 And an iOS:
<img width="367" alt="screen shot 2016-10-23 at 9 02 59 pm" src="https://cloud.githubusercontent.com/assets/2053909/19631031/1c3ec9f0-9964-11e6-980c-bb922e1a7250.png">

Removing the classes displays the image

```
<ListView items="{{ items }}" itemTap="onItemTap">
  <ListView.itemTemplate>
    <StackLayout orientation="horizontal">
      <Image src="~/images/test.png" />
      <Label text="Some text..." />
    </StackLayout>
  </ListView.itemTemplate>
</ListView>
```

After removing classes: (iOS only shown below)
<img width="370" alt="screen shot 2016-10-23 at 9 06 13 pm" src="https://cloud.githubusercontent.com/assets/2053909/19631052/934be546-9964-11e6-9df7-545dc1022b2b.png">

**Comment by insic** [comment]
I experienced the same issue as well. And the bug can be seen also on the "NativeScript 2.3 - Angular 2 Awakening!" video https://youtu.be/WQktviwAGb0?t=46m10s

**Comment by sotisoti** [comment]
Same here,

My tns info
```
All NativeScript components versions information
┌──────────────────┬─────────────────┬────────────────┬───────────────┐
│ Component        │ Current version │ Latest version │ Information   │
│ nativescript     │ 2.3.0           │ 2.3.0          │ Up to date    │
│ tns-core-modules │ 2.3.0           │ 2.3.0          │ Up to date    │
│ tns-android      │ 2.3.0           │ 2.3.0          │ Up to date    │
│ tns-ios          │                 │ 2.3.0          │ Not installed │
└──────────────────┴─────────────────┴────────────────┴───────────────┘
```

**Comment by sotisoti** [comment]
I found a solution, just need to make sure your package.json has the latest version of tns-core-modules:
```
"tns-core-modules": "next"
```

**Comment by NathanWalker** [comment]
This has been fixed with `2.4.0`. @mikebranstein if you get a chance to verify by updating, could you close this if it works for you now?

**Comment by NathanWalker** [comment]
Demo app here updated to `2.4.0` which fixes this. If anyone comes across this experiencing same issue, please update to tns `2.4.0`.
